components/Logo.tsx

```diff
@@ -0,0 +1,15 @@
+import Link from 'next/link';
+import Image from '/components/Image';
+import styles from '../styles/components/Logo.module.scss';
+
+interface LogoProps {
+  href?: string;
+}
+
+export default function Logo({ href = '/' }: LogoProps) {
+  return (
+    <Link href={href} className={styles.logo}>
+      <Image src='/logo.svg' alt='TCSCA Logo' width={400} height={400} />
```

Comment:
This pr looks great, only thing is can we use the `fill` attribute on this and have the logo size to whatever the parent container is this will make it way more useful on the website, as the herobanner logo might be like `400*400` but the navbar will not be, we can also additionally pass in the `width` and `height`, just to help with first paint but given its an svg the width and height wont allow us todo image optimization and load the proper sized image.